hadoop-ozone/recon/src/main/java/org/apache/hadoop/ozone/recon/fsck/ContainerHealthTask.java

```diff
@@ -108,6 +97,30 @@ public synchronized void run() {
     }
   }
 
+  public void triggerContainerHealthCheck() {
+    try {
+      lock.writeLock().lock();
```

Comment:
We typically `lock()` right above `try`:
```suggestion
    lock.writeLock().lock();
    try {
```

Reply:
@smengcl - Thanks for your review. It is fixed as per your suggestion. Pls re-review and merge.